src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import 'variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar__day {
+  box-sizing: border-box;
+  margin: 0;
+  height: $width;
+  width: $width;
+  background-color: #eee;
+  border: 1px solid black;
+  transition: transform 0.5s, background-color 0.5s;
+  &:hover {
+    background-color: #FFBFCB;
+    transform: translateY($slide);
+  }
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  flex-flow: row wrap;
+  width: 7 * ($width + $gap);
+  margin: auto;
+  line-height: $width - 2;
+  text-align: center;
+
+  @for $number from 1 through 31 {
+    & :nth-child(#{$number}) {
+      &::before {
+        content: "#{$number}";
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+        padding-left: 1px;
```

Comment:
```suggestion

```
Consider applying these styles for the element once instead of generating the same styling for each nth-child in the cycle